docs/kb/rules/run-all-available-rules.md

```diff
@@ -34,17 +40,20 @@ This command will pull out all available public, org private, and pro tiered rul
                                                                                                                         
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:02:31   
 ```
-The above example output is when you are running `--config=r/all` while logged in via `semgrep ci`. If you are not logged in you will just use the public rules within `r/all`.
 
-However there is one error that you should be wary about which is the following:
+If you are not logged in, `config=r/all` runs public rules only.
+
+## Troubleshooting invalid configuration file
+
+The following error may occur:
 
 ```
 [ERROR] invalid configuration file found (1 configs were invalid)
 ```
 
-This error may come up when running `r/all` if there is a current error within any of your custom/private rules.
+This error appears if there is a syntax or other error within any of your custom rules.
 
-To get around this you can use the following two commands as a workaround:
+Enter the following two commands as a workaround:
```

Comment:
IT might be worth it to explain to users what the commands do so that they don't think they're arbitrarily deleting all their custom rules.

Reply:
<img width="414" alt="Snag_19e6d52" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/bb77f591-e73f-4811-9184-f5375a82a3ec">

See above for explanation for what the commands do.